assets/i18n/en.json

```diff
@@ -3,7 +3,7 @@
     "I18N_ACTION_APPLY_TO_TEACH_WITH_OPPIA": "Apply to Teach With Oppia",
     "I18N_ACTION_BROWSE_EXPLORATIONS": "Browse our Explorations",
     "I18N_ACTION_BROWSE_LESSONS": "Browse our Lessons",
-    "I18N_ACTION_CREATE_EXPLORATION": "Create your own Exploration",
+    "I18N_ACTION_CREATE_EXPLORATION": "Create an Exploration",
```

Comment:
Hi @Nikerabbit @siebrand, a quick question: if we change a string in en.json (the source file), will translatewiki pick this up automatically and note that the corresponding strings for the other languages need to be changed, or do we have to notify translatewiki in some other way? I know that new strings are picked up, but I don't think we've had a case before where the English text for a particular identifier got changed.